Backend/socket.js

```diff
@@ -0,0 +1,14 @@
+let io;
```

Comment:
I don't know, maybe we should make a stateful components instead?

Reply:
Statefull components are in the reactjs not in the server side (Nodejs) here all files are backend files, so we keep it as it is.

Reply:
I mean, high order functions. I think it will be more testable in that case.

```
const module = (knex) => () => ({
  method: async (arg) => {
     await knex.doSomething(arg)
  }
})

```

Reply:
where do you want higher order functions?